documentation/src/main/doc/modules/kafka/pages/protobuf-configuration.adoc

```diff
@@ -0,0 +1,45 @@
+[#kafka-protobuf-configuration]
+== Using Google Protobuf serializer/deserializer
+
+If you are using https://developers.google.com/protocol-buffers/[Protocol Buffers] serializer/deserializer, please note the following configuration properties.
+
+=== For https://docs.confluent.io/platform/current/schema-registry/serdes-develop/serdes-protobuf.html#serdes-and-formatter-protobuf[Confluent] Schema Registry
```

Comment:
I don't believe we support links as headers (as the header is a link itself).

Reply:
Can you add a link in the introduction sentence?

Reply:
We have the same in the Avro guide, so should be fine.